packages/web/src/scss/components/FileUploader/_FileUploaderInput.scss

```diff
@@ -50,13 +51,10 @@ $_field-name: 'FileUploaderInput';
 
 // 1.
 .FileUploaderInput__link {
-    cursor: pointer;
+    @include links.base();
+    @include links.stretch();
 
-    &::before {
-        content: '';
-        position: absolute;
-        inset: 0;
-    }
+    cursor: pointer;
```

Comment:
Not quite that… `cursors.$button-links` is meant to override the default cursor of `<button>` element. Maybe the variable name is misleading, thanks for pointing it out!

Reply:
Variable name changed to `$buttons`.

We set it here directly because `.FileUploaderInput__link` is a span that emulates link. And links will always have `cursor: pointer`, no matter any settings.